src/main/java/com/epam/ta/reportportal/core/events/activity/IntegrationCreatedEvent.java

```diff
@@ -56,6 +65,7 @@ public Activity toActivity() {
 				.addObjectId(integrationActivityResource.getId())
 				.addObjectName(integrationActivityResource.getTypeName())
 				.addProjectId(integrationActivityResource.getProjectId())
+				.addHistoryField(Optional.of(integrationNameField))
```

Comment:
can be used 'addHistoryField(String field, String before, String after)' here, so we can remove upper above

Reply:
fixed